src/Glpi/Form/Destination/CommonITILField/ITILCategoryField.php

```diff
@@ -147,7 +147,7 @@ public function convertFieldConfig(FormMigration $migration, Form $form, array $
                     specific_question_id: $migration->getMappedItemTarget(
                         'PluginFormcreatorQuestion',
                         $rawData['category_question']
-                    )['items_id'],
+                    )['items_id'] ?? 0
```

Comment:
It means that the target question does not exists, IMHO, it should trigger an error, not be ignored silently.
The same remark is valid for all other similar occurences.